### DISCUSSION TITLE
Tracking training courses for OSPO activities

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#53] [Community Proposals]
OSPO activities cover a wide range of fields and tasks (e.g managing open source licensing agreements, ensuring open source compliance, setting policy and processes, improving developer education and outbound contributions from developers, building an effective open source strategy, and more).

This discussion thread welcomes the open source community to put together a list of *existing open source training courses* to help OSPO managers and teams' open source education.

To give more structure to these courses, contributors can group training courses using any of these fields:

* OS Strategy and Processes
* OS Security and Compliance
* OS Community Management
* OS Project Management

**Comment by anajsana** [comment]
Some resources:

## OS Security and Compliance

[Introduction to Open Source License Compliance Management (LFC193) - Free](https://training.linuxfoundation.org/training/introduction-to-open-source-license-compliance-management-lfc193/)
[Generating Software Bill of Materials - Free](https://training.linuxfoundation.org/training/generating-a-software-bill-of-materials-sbom-lfc192/)
[Secure Software Development Requirements: Design and Reuse - Free](https://training.linuxfoundation.org/training/secure-software-development-requirements-design-and-reuse-lfd104/)
[Secure Software Development: Implementation - Free](https://training.linuxfoundation.org/training/secure-software-development-implementation-lfd105/)
[Open Source Licensing Basics for Software Developers - Free](https://training.linuxfoundation.org/training/open-source-licensing-basics-for-software-developers/)

## OS Strategy and Processes

[Open Source Management and Strategy - Paid](https://training.linuxfoundation.org/training/open-source-management-and-strategy/)
[OSPO 101 - Free](https://github.com/todogroup/ospo101)

## OS Community Management

[Inclusive Strategies for Open Source - Free](https://training.linuxfoundation.org/training/inclusive-strategies-for-open-source-lfc103/)

**Comment by Raghavendra-BJ** [comment]
Strategy/management
Fundamentals of professional open source management- Free (LFC-110)
https://training.linuxfoundation.org/training/fundamentals-of-professional-open-source-management-lfc110/

**Comment by semioticrobotic** [comment]
[Establish an Open Source Program Office](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html) - Paid
Part of the Open Source Technology Management Program at Brandeis University
(OS Strategy and Process)

**Reply by Masson-Apereo** [reply]
Here's the link to Brandeis University's full certificate program, [Open Source Technology Management program](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/index.html).

Courses Offered

- [Cultivate an Open Source Community](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#community)
- [Integrating Open Source Communities in Corporate Environments](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#integrating)
- [Open Source Workflow and Infrastructure](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#infrastructure)
- [Production of Distributed Open Source Software](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#production)
- [Open Source Business Practices](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#business-practices)
- [Establish an Open Source Program Office](https://www.brandeis.edu/gps/professional-development/for-professionals/ostm/courses.html#office)

**Comment by emmairwin** [comment]
Two courses we published to GitHub (meant to be platform agnositic, and organized like a Jekyll site)

https://github.com/microsoft/OSPO-Courses

- [Inclusive Open Source Governance, Code of Conduct Enforcement](https://github.com/microsoft/OSPO-Courses/tree/main/inclusive-governance)
- [Mapping your Open Source Career](https://github.com/microsoft/OSPO-Courses/tree/main/mapping-your-oss-career)

**Comment by emmairwin** [comment]
@anajsana does TODO group have any plans host courses at some point? 
I am only asking as we had a contributor interested in exploring how we might publish the courses I linked above  on our opensource.microsft.com page, which would be awesome but also like to see how we can collaborate with what others are doing .

**Comment by caniszczyk** [comment]
We have course material we open sourced and shared here:
https://github.com/todogroup/ospo101 which essentially is this course
https://training.linuxfoundation.org/training/open-source-management-and-strategy/

We can definitely hosted courses via LFT if you wanted to do something
along those lines

What's your goal :)?

On Tue, May 10, 2022 at 3:17 PM Emma ***@***.***> wrote:

> @anajsana <https://github.com/anajsana> does TODO group have any plans
> host courses at some point?
> I am only asking as we had a contributo interested in exploring how we
> might publish the courses I linked above (which are just in markdown) on
> our opensource.microsft.com page, which would be awesome but also like to
> see how we can collaborate with what others are doing .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/todogroup/ospology/discussions/53#discussioncomment-2725478>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPSII4ESYL66M37F2NGNLVJK74VANCNFSM5N2HOACQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Cheers,

Chris Aniszczyk
https://aniszczyk.org
